bulbs/infographics/enum.py

```diff
@@ -0,0 +1,9 @@
+from django_enumfield import enum
+
+
+class InfographicType(enum.Enum):
```

Comment:
There are circular dependency issues regarding utility/validation between rest_framework + models.py that prohibit that. +++++ I think custom fields should be moved out of models anyway